index.html

```diff
@@ -76,5 +76,195 @@ <h1>
+
+      <!-- second card -->
+
+      <div class="card card-1">
+        <div class="left">
+          <img src="assets/snapshot-2.png" alt="" class="snap" />
+        </div>
+        <div class="right">
+          <h1>Multi-Post Stories</h1>
+          <div class="canopy">
+            <h4>Canopy</h4>
+            <img src="assets/circle.png" alt="" />
+            <h4>Back End Dev</h4>
+            <img src="assets/circle.png" alt="" />
+            <h4>2015</h4>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+
+          <button class="button" type="button">See Project</button>
+        </div>
+      </div>
+
+      <!-- third card -->
+
+      <div class="card card-1">
+        <div class="left">
+          <img src="assets/snapshot-3.png" alt="" class="snap" />
+        </div>
+        <div class="right">
+          <h1>Tonic</h1>
+          <div class="canopy">
+            <h4>Canopy</h4>
+            <img src="assets/circle.png" alt="" />
+            <h4>Back End Dev</h4>
+            <img src="assets/circle.png" alt="" />
+            <h4>2015</h4>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <button class="button" type="button">See Project</button>
+        </div>
+      </div>
+
+      <!-- fourth card -->
+
+      <div class="card card-1">
+        <div class="left">
+          <img src="assets/snapshot-4.png" alt="" class="snap" />
+        </div>
+        <div class="right">
+          <h1>Multi-Post Stories</h1>
+          <div class="canopy">
+            <h4>Canopy</h4>
+            <img src="assets/circle.png" alt="" />
+            <h4>Back End Dev</h4>
+            <img src="assets/circle.png" alt="" />
+            <h4>2015</h4>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <button class="button" type="button">See Project</button>
+        </div>
+      </div>
+    </section>
+
+    <section class="about-section">
+      <div class="about-me">
+        <h1>About Myself</h1>
+        <p>
+          Hello I&#39;m a software developer! I can help you build a product ,
+          feature or website. Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don&#39;t
+          hesitate to contact me.
+        </p>
+        <p class="connect">LET&#39;S CONNECT</p>
```

Comment:
- [ ] I kindly suggest that it would be better to use a **heading** tag, e.g ``h3`` or ``h2`` because **LETS CONNECT** is not a paragraph and it is important to use the right tags for each content 😉